docs/02-for-app-authors/10-verification.md

```diff
@@ -2,7 +2,30 @@
 
 ## What is verification?
 
-Verification is a process by which Flathub can confirm that an app is published by its original developer or a third party approved by the developer. This is done by verifying the developers identity using a link to their website or to their profile on a source code hosting site.
+Verification is a process by which Flathub can confirm that an app is published by its original developer or a third party approved by the developer. This is done by verifying the developer's identity using a link to their website or to their account on a source code hosting site. The methods available for verification depend on the app ID.
+
+### Well-known URI
+
+Flathub may use the app ID in reverse domain name notation to determine where to look for a well-known URI. For example, an app ID `com.example.App` would require verifying the `example.com` domain with a response at `https://example.com/.well-known/org.flathub.VerifiedApps.txt` containing the unique code provided by the Flathub website during the verification process.
+
+For convenience and flexibility, verification of the well-known URI will follow redirects.
+
+### Source code hosting site
+
+Apps with IDs indicating supported source code hosting sites like GitHub, GitLab, GNOME GitLab, and KDE GitLab may be verified by authenticating with Flathub using an authorized account. The following app ID prefixes will be matched:
+  
+  - **GitHub**: `io.github.`[^1]
+  - **GitLab**: `io.gitlab.`[^1]
+  - **GNOME GitLab**: `org.gnome.`
+  - **KDE GitLab**: `org.kde.`
+
+For GitHub, the authenticated account must be the owner of the repository. If the repository is owned by an organization, the account must be an admin of the organization.
+
+For GitLab, the authenticated account must be the owner of the repository. If the repository is owned by a group, the account must be an owner, maintainer, or developer of that group.
+
+For GNOME and KDE apps, additional checks may be performed to ensure the account is authorized to perform verification for that app.
```

Comment:
Was going to remark on this, but I guess they should/will just reach out and are not really the target of the doc here